src/DGtal/helpers/ShortcutsGeometry.h

```diff
@@ -976,6 +981,132 @@ namespace DGtal
           return mc_estimations;
         }
 
+      
+        /// Given a digital shape \a bimage, a sequence of \a surfels,
+        /// and some parameters \a vm, returns the principal curvatures and
+        /// directions using an Integral
+        /// Invariant (II) estimation at the specified surfels, in the
+        /// same order.
+        ///
+        /// @param[in] bimage the characteristic function of the shape as a binary image (inside is true, outside is false).
+        /// @param[in] surfels the sequence of surfels at which we compute the Gaussian curvatures
+        /// @param[in] params the parameters:
+        ///   - verbose         [     1]: verbose trace mode 0: silent, 1: verbose.
+        ///   - r-radius        [   3.0]: the constant for kernel radius parameter r in r(h)=r h^alpha (VCM,II,Trivial).
+        ///   - alpha           [  0.33]: the parameter alpha in r(h)=r h^alpha (VCM, II)."
+        ///   - gridstep        [   1.0]: the digitization gridstep (often denoted by h).
+        ///
+        /// @return the vector containing the estimated Gaussian curvatures, in the
+        /// same order as \a surfels.
+        ///
+        /// @note The function is faster when surfels are in a specific order, as
+        /// given for instance by a depth-first traversal (@see getSurfelRange)
+        static CurvatureTensorQuantities
+          getIIPrincipalCurvaturesAndDirections( CountedPtr<BinaryImage> bimage,
+                                                const SurfelRange&      surfels,
+                                                const Parameters&       params
+                                                = parametersGeometryEstimation()
+                                                | parametersKSpace() )
+        {
+          auto K =  getKSpace( bimage, params );
+          return getIIPrincipalCurvaturesAndDirections( *bimage, K, surfels, params );
+        }
+
+      /// Given a digital shape \a bimage, a sequence of \a surfels,
+      /// and some parameters \a vm, returns the principal curvatures and
+      /// directions using an Integral
+      /// Invariant (II) estimation at the specified surfels, in the
+      /// same order.
+      ///
+        /// @param[in] dshape the digitized implicit shape, which is an
```

Comment:
Change of indentation width detected :octocat:

Reply:
👍